int64 traj_id
time start_time

bool have_retiming

int32 order_3d
float64[] knots_3d
geometry_msgs/Point[] pos_pts_3d

int32 order_1d
float64[] knots_1d
float64[] pos_pts_1d

int32 yaw_strategy
geometry_msgs/Point yaw_dir_pt

int32 order_yaw
float64[] knots_yaw
float64[] yaw_pts